settings.gradle.kts

```diff
@@ -25,9 +25,13 @@ plugins {
 }
 
 include(
-    ":kotlinpoet",
-    ":interop:javapoet",
-    ":interop:kotlinx-metadata",
-    ":interop:ksp",
-    ":interop:ksp:test-processor",
+  ":kotlinpoet",
+  ":interop:javapoet",
+  ":interop:kotlinx-metadata",
+  ":interop:ksp",
+  ":interop:ksp:test-processor",
 )
+
+rootProject.name = "kotlinpoet-parent"
```

Comment:
Gradle doesn't like having two projects with the same name.

Reply:
Sounds good, updated! [f2f738c](https://github.com/square/kotlinpoet/pull/1602/commits/f2f738c08bd49ab49e5443b941dcd1964095ba19)